interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -1084,6 +1085,10 @@ namespace cling {
     utils::Analyze::maybeMangleDeclName(FD, mangledNameIfNeeded);
     IncrementalExecutor::ExecutionResult ExeRes =
        m_Executor->executeWrapper(mangledNameIfNeeded, res);
+
+    // Force-flush as we might be printing on screen with printf.
+    std::cout.flush();
```

Comment:
This happens to also flush by `stdout` on most implementations, but I think it's not required? Should we add `fflush(stdout)`?